cg/store/models.py

```diff
@@ -219,7 +219,7 @@ class Analysis(Model):
     family_id = Column(ForeignKey("family.id", ondelete="CASCADE"), nullable=False)
     uploaded_to_vogue_at = Column(types.DateTime, nullable=True)
 
-    family = orm.relationship("Family", back_populates="analyses")
+    family = orm.relationship("Case", back_populates="analyses")
```

Comment:
```suggestion
    case = orm.relationship("Case", back_populates="analyses")
```

Reply:
Yeah, leaving it for a separate PR though